core/src/main/java/zipkin2/reporter/Reporter.java

```diff
@@ -24,16 +22,16 @@
  * zipkin2.Span).
  */
 public interface Reporter<S> {
-  Reporter<Span> NOOP = new Reporter<Span>() {
-    @Override public void report(Span span) {
+  Reporter<zipkin2.Span> NOOP = new Reporter<zipkin2.Span>() {
```

Comment:
The change in this file appears to be a no-op, not sure if it's intended. Import statements are syntax sugar but don't think they have any effect on bytecode

Reply:
this was a relic from when I removed the dep entirely, which broke too much. reverted!